rules/macos/execution_shell_execution_via_apple_scripting.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the Shell process via Apple Scripting (JXA or AppleScript). Adversaries may use
+doShellScript functionality in JXA or do shell script in AppleScript to execute System commands.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Shell Execution via Apple Scripting"
```

Comment:
```suggestion
name = "Shell Execution via Scripting"
```

Reply:
prefer apple scripting, scripting includes also shell scripts which are not covered by this rule